include/swift/Basic/OptionSet.h

```diff
@@ -58,6 +59,10 @@ class OptionSet {
   /// Create an option set with only the given option set.
   constexpr OptionSet(Flags flag) : Storage(static_cast<StorageType>(flag)) {}
 
+  /// Create an option set containing the given options.
+  constexpr OptionSet(std::initializer_list<Flags> flags)
```

Comment:
This constructor cannot be `constexpr` unless `combineFlags` is `constexpr`.

Reply:
Huh, of course. Done.

I'm surprised the compiler didn't complain about this?